templates/rapidsai-core/partials/patch.dockerfile.j2

```diff
@@ -3,3 +3,9 @@
 {# Patch for CVE-2020-8116 https://github.com/advisories/GHSA-ff7x-qrg7-qggm #}
 RUN source activate rapids \
     && npm i -g npm@">=7"
+
+{% if "centos" in os %}
+RUN yum -y upgrade
+{% else %}
+RUN apt-get update && apt-get -y upgrade
```

Comment:
```suggestion
RUN yum -y upgrade \
    && yum clean all
{% else %}
RUN apt-get update \
    && apt-get -y upgrade \
    && rm -rf /var/lib/apt/lists/*
```

Cleans up `apt` and `yum` cache to reduce image size.

src: 
- https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get
- https://hub.docker.com/_/centos